core/src/main/java/org/apache/iceberg/BaseFilesTable.java

```diff
@@ -166,7 +166,6 @@ public CloseableIterable<StructLike> rows() {
       if (readableMetricsField == null) {
         return CloseableIterable.transform(files(projection), file -> (StructLike) file);
       } else {
-
```

Comment:
Nit: revert this change?

Reply:
oh yea it is definitely subjective, i remember vaguely Anton suggest some newlines after else for better readability at some point for my prs (not sure for this particular line or not).  Id say just remove it to minimize the change?

Reply:
thank you for the context, reverted!